docs/development/code.md

```diff
@@ -37,3 +37,32 @@ without having to write boilerplate code.
 processing them in the background with workers.
 
 It will be used to process the jobs that will be created by the runner manager.
+
+## Typing
+
+`pyright` configuration is located in the
+[`pyrightconfig.json`](../../pyrightconfig.json) file.
+
+It is run automatically when `trunk check` is run.
+
+### Stub files
```

Comment:
Explain what this module is used for.

Reply:
It's written just below, any particular question?